src/components/Home/WhatsNewModal/index.tsx

```diff
@@ -91,12 +91,14 @@ const WhatsNewModal: React.FC = () => {
               <h2>{latestPost.title}</h2>
             </Link>
             <Link href={latestPost.slug}>
-              <GatsbyImage
-                className={styles.image}
-                alt=""
-                objectFit="contain"
-                image={latestPost.picture.gatsbyImageData}
-              />
+              {latestPost.picture && (
+                <GatsbyImage
+                  className={styles.image}
+                  alt=""
+                  objectFit="contain"
+                  image={latestPost.picture.gatsbyImageData}
+                />
+              )}
             </Link>
```

Comment:
Should we put the conditional on the `Link` component so we don't render an empty link?

Reply:
Nice catch! I'll update the code!